backend/internal/supabase/migrations/20241022174417_drop_one_review_per_user_constraint.sql

```diff
@@ -0,0 +1 @@
+alter table review drop constraint unique_user_media;
```

Comment:
no longer limiting users to one review per user per song/album

put it in this PR since it made testing way easier